packages/core/src/__tests__/__e2e__/switch/Switch.cy.tsx

```diff
@@ -175,4 +175,21 @@ describe("GIVEN a Switch", () => {
     cy.findByRole("switch").should("be.focused");
     cy.findByRole("switch").should("be.checked");
   });
+
+  describe("WHEN readOnly is true", () => {
+    it("THEN should not toggle if clicked", () => {
+      cy.mount(<Switch readOnly />);
+      cy.findByRole("switch").should("not.be.checked");
+      cy.findByRole("switch").realClick();
+      cy.findByRole("switch").should("not.be.checked");
+    });
+
+    it("THEN should not toggle when pressing the Space key", () => {
+      cy.mount(<Switch readOnly />);
+      cy.findByRole("switch").should("not.be.checked");
```

Comment:
no test of focus but it's something we call out as a differentiator between disabled?